### PR TITLE
[FEATURE][MER-2086] Support adding unordered pages to sections and products

### PR DESCRIPTION
**Pull request by augusto1024**
[Link to the ticket](https://eliterate.atlassian.net/browse/MER-2086)

https://github.com/Simon-Initiative/oli-torus/assets/53446634/f0011508-70e2-4ca0-b516-960628805038

### CODE REVIEW

File: assets/css/table.css [hunk 1]
```diff
@@ -60,10 +65,23 @@
...
+  @apply overflow-x-hidden;
+}
+
+.remix_materials_table {
+  @apply instructor_dashboard_table border-none;
+}
+
+.remix_materials_table th:last-of-type {
+  text-align: right;
+>>>>>>> e6ee73b5a5 (Add ability to add orphan pages to section curriculum)
```
Comment (simonchoxx) [inline assets/css/table.css:86]: If I'm not mistaken, on lines 75 and 86 is the text that VSCode shows when conflicts need to be resolved, I'm right?

File: lib/oli/delivery/sections.ex [hunk 2]
```diff
@@ -2420,18 +2422,68 @@ defmodule Oli.Delivery.Sections do
...
+      else
+        nil
+      end
+
+    offset =
+      if params[:offset] do
+        params.offset
+      else
+        0
+      end
```
Comment (simonchoxx) [inline lib/oli/delivery/sections.ex:2445]: Maybe in `if` statements you could use the short way of writing it, something like: limit = if params[:limit], do: params.limit, else: nil , WDYT?

File: lib/oli/publishing.ex [hunk 3]
```diff
@@ -675,6 +676,89 @@ defmodule Oli.Publishing do
...
+      else
+        nil
+      end
+
+    offset =
+      if params[:offset] do
+        params.offset
+      else
+        0
+      end
```
Comment (simonchoxx) [inline lib/oli/publishing.ex:716]: Same as above

File: lib/oli_web/live/delivery/remix_section.ex [hunk 4]
```diff
@@ -501,6 +527,34 @@ defmodule OliWeb.Delivery.RemixSection do
...
+          selected_publication: publication
+        } = modal_assigns
+    } = socket.assigns
+
+    item = Enum.find(table_model.rows, &(&1.uuid == uuid))
+
+    modal_assigns = %{
+      modal_assigns
+      | selection:
+          xor(
```
Comment (simonchoxx) [inline lib/oli_web/live/delivery/remix_section.ex:549]: I have no idea what this `xor` thing is hahaha, are you doing some kind of magic here?? 😂😂.

**Review by simonchoxx — APPROVED** [review]
Nice work mate!! 🚀🚀🚀

File: assets/css/table.css (hunk 1 above)
Comment (augusto1024) [reply assets/css/table.css:86]: Yeah, I don't know how I missed that 🤔

File: lib/oli_web/live/delivery/remix_section.ex (hunk 4 above)
Comment (augusto1024) [reply lib/oli_web/live/delivery/remix_section.ex:549]: Actually I just copied it from another place. The function is defined below, in line `755`. What it does is, if the tuple that is being added is already in the list, it removes it, if not, it adds it

File: assets/css/table.css [hunk 5]
```diff
@@ -60,10 +65,23 @@
   @apply bg-red-200 bg-opacity-20;
 }
 
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (augusto1024) [reply assets/css/table.css:68]: Yeah, I don't know how I missed that 🤦

**Review by eliknebel — APPROVED** [review]

**Pull request merged**